### ISSUE TITLE
help with error

### ISSUE THREAD

**Issue by bnglomics** [RasmussenLab/MOVE#99]
getting this error after running the first command.  I triple checked the directories and all the files seem to be in place...

FileNotFoundError: [Errno 2] No such file or directory: 'interim_data/random.small.drugs.npy'

**Comment by enryH** [comment]
So this is after this step ([link](https://move-dl.readthedocs.io/tutorial/introduction.html#encoding-data)? Could you try in the [colab tutorial](https://colab.research.google.com/drive/1RFWNsuGymCmppPsElBvDuA9zRbGskKmi?usp=sharing)?

**Comment by bnglomics** [comment]
Yes, that's the one that didn't work.  thanks, I'll try the colab tutorial. 

Any fixes for that error that I can try?

**Comment by enryH** [comment]
Hmm. So you cloned the repository and your current pwd is the tutorial folder? I would test it in a GitHub codespace starting from the repository if I were you. then you are sure it is not due to some setup on your local machine/cluster.

**Comment by bnglomics** [comment]
Thanks!  it was a PyCharm issue and I'm all set!
